kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -893,6 +978,28 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  private def getMetadata(engine: Engine, table: Table, version: Long): Metadata = {
```

Comment:
[nit] this name is slightly misleading. This is the metadata action from commit version X. Maybe `getMetadataActionFromCommit` is better?